### PR TITLE
Fix MSRV CI to correctly use MSRV

### PR DESCRIPTION
**Pull request by davidhewitt**
I think without this setting the specified Rust version does not become the default, and I think #1204 may have broken MSRV support 😢

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -78,12 +78,14 @@ jobs:
...
           profile: minimal
+          default: true
 
       - if: matrix.platform.os == 'ubuntu-latest'
         name: Prepare LD_LIBRARY_PATH (Ubuntu only)
         run: echo LD_LIBRARY_PATH=${pythonLocation}/lib > $GITHUB_ENV
 
-      - run: rustup set default-host ${{ matrix.platform.rust-target }}
+      - name: Prepare CARGO_BUILD_TARGET
+        run: echo CARGO_BUILD_TARGET=${{ matrix.platform.rust-target }} > $GITHUB_ENV
```
Comment (alex) [inline .github/workflows/ci.yml:88]:
```suggestion
        run: echo CARGO_BUILD_TARGET=${{ matrix.platform.rust-target }} >> $GITHUB_ENV
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -78,12 +78,14 @@ jobs:
           toolchain: ${{ matrix.rust }}
           target: ${{ matrix.platform.rust-target }}
           profile: minimal
+          default: true
 
       - if: matrix.platform.os == 'ubuntu-latest'
         name: Prepare LD_LIBRARY_PATH (Ubuntu only)
         run: echo LD_LIBRARY_PATH=${pythonLocation}/lib > $GITHUB_ENV
```
Comment (alex) [inline .github/workflows/ci.yml:85]:
```suggestion
        run: echo LD_LIBRARY_PATH=${pythonLocation}/lib >> $GITHUB_ENV
```

**Review by alex — COMMENTED** [review]
Just noticed this. Sorry!

**Comment by davidhewitt** [comment]
Thanks, that would have had me stumped 😅

**Comment by kngwyu** [comment]
Thank you for the fix!

**Pull request merged**